src/library/scala/collection/immutable/Range.scala

```diff
@@ -400,18 +400,17 @@ extends scala.collection.AbstractSeq[Int]
    *  equals contract.
    */
 
-  override def toString() = {
-    val endStr =
-      if (numRangeElements > Range.MAX_PRINT || (!isEmpty && numRangeElements < 0)) ", ... )" else ")"
-    take(Range.MAX_PRINT).mkString("Range(", ", ", endStr)
+  override def toString = {
+    val preposition = if (isInclusive) "to" else "until"
+    f"$start $preposition $end${
+      if (step == 1) "" else f" by $step"
+    }"
```

Comment:
Is there any reason to use `f` instead of `s`? `f` will pull in a lot of code in Scala.js code bases that do not otherwise use `String.format`.

Also, I'm not convinced the nested interpolator is more readable than:
```scala
s"$start $preposition $end" + (if (step == 1) "" else s" by $step")
```